pyproject.toml

```diff
@@ -191,10 +193,9 @@ impala = ["fsspec", "impyla", "requests", "sqlalchemy"]
 mssql = ["sqlalchemy", "pymssql", "sqlalchemy-views"]
 mysql = ["sqlalchemy", "pymysql", "sqlalchemy-views"]
 pandas = ["regex"]
-polars = ["polars", "pyarrow"]
+polars = ["polars"]
 postgres = ["psycopg2", "sqlalchemy", "sqlalchemy-views"]
-pyarrow = ["pyarrow"]
```

Comment:
I'm going to remove this before 5.1 so we don't introduce it, only to take it away in a major release.